### PR TITLE
Partner details page fixes

### PR DESCRIPTION
**Pull request by adi130987**
This PR will solve issues: 7, 22, 48, 49 and 53 from issues list.

### CODE REVIEW

File: pmp/app-elements/partners/partner-details.html [hunk 1]
```diff
@@ -274,92 +261,38 @@
...
             this.contentVisible = true;
-            // check & set shared_partner values
-            this._setSharedPartnerValues();
 
             // decide if we should show core values assessment attachment
             this.set('showCoreValuesAssessmentAttachment',
                 this._showCoreValueAssessment(partner.partner_type, partner.cso_type, partner.core_values_assessment_file));
             this.set('originalPartnerData', JSON.parse(JSON.stringify(partner)));
           } else {
+            // TODO: @rob ... what purpose has this contentVisible property?
```
Comment (robertavram) [inline pmp/app-elements/partners/partner-details.html]: can be removed

File: pmp/app-elements/partners/partner-details.html [hunk 2]
```diff
@@ -371,46 +304,30 @@
...
-              if (sharedPartners !== partner.shared_with) {
-                console.log(sharedPartners, partner.shared_with);
-                changes[fieldName] = sharedPartners;
+              if ((partner.hasOwnProperty('core_values_assessment')) &&
+                (partner.core_values_assessment instanceof File)) {
+                  changes.core_values_assessment = partner.core_values_assessment;
               }
             } else {
-              if (partner[fieldName] !== this.originalPartnerData[fieldName]) {
+              if (JSON.stringify(partner[fieldName]) !== JSON.stringify(this.originalPartnerData[fieldName])) {
```
Comment (robertavram) [inline pmp/app-elements/partners/partner-details.html]: isn't there a good way of comparing this with underscore?
if this works ok.. it's fine.. but i'm concerned about certain values not being able to be stringified... thoughts?

**Comment by adi130987** [comment]
Issues from the list fixed in this PR:

* 7, 72 - used esmm dropdown dynamic-align to fix
* 22 - remove toats errors from partner details and used etools-error-messages-box new element to show the errors
* 42, 48, 49, 53, 54 - fixed

**Comment by adi130987** [comment]
New fixes added for list issues:

57 - Issue#2 fixed
60 - staff members title field not required 
85 - etools-single-file fix

**Pull request merged**